### PR TITLE
[master] Fix CS

### PR DESCRIPTION
**Pull request by SpacePossum**
After the last merges the CS of the Fixer itself is off.

### CODE REVIEW

File: Symfony/CS/Tests/Fixer/Symfony/MethodSeparationFixerTest.php [hunk 1]
```diff
@@ -401,8 +400,7 @@ protected function getE(){echo 3;}
 private $a;
 
 function getF(){echo 4;}
-}'
-        , '<?php
+}', '<?php
```
Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/MethodSeparationFixerTest.php]: I don't like that...

**Comment by keradus** [comment]
Thank you for catching this. I will check all branches and fix what is needed.

**Comment by keradus** [comment]
Thank you @SpacePossum

**Comment by SpacePossum** [comment]
:+1: thank you @keradus :)

**Pull request closed without merging**